ui/pages/confirmations/hooks/alerts/transactions/useInsufficientBalanceAlerts.ts

```diff
@@ -48,7 +48,12 @@ export function useInsufficientBalanceAlerts(): Alert[] {
     selectTransactionFeeById(state, transactionId),
   );
 
-  const nativeCurrency = useSelector(getMultichainNativeCurrency);
+  const [multichainNetworks, evmNetworks] = useSelector(
+    getMultichainNetworkConfigurationsByChainId,
+  );
+  const nativeCurrency = (
+    multichainNetworks[chainId as CaipChainId] ?? evmNetworks[chainId]
```

Comment:
Is this alert even used in Solana? Are these alerts not exclusive to EVM transactions? In which case should we use `selectNetworkConfigurationByChainId` instead?

Reply:
yep good point, here we can use only evm chains, let me optimise this in following PR